test/BUILD.gn

```diff
@@ -492,6 +492,11 @@ test("brave_unit_tests") {
     deps += [ "//brave/browser/ui/webui/brave_education:unit_tests" ]
   }
 
+  # Theme command line switches are desktop-only
+  if (is_win || is_mac || is_linux || is_chromeos) {
```

Comment:
Just put this in the `!is_android` block, as we do not concerns ourselves with `is_chromeos`, and nothing in this gn file gets added to `is_ios` builds.